commit d0b8329cd69bef1abf341140a933cd91e1cdfbdf (HEAD -> t1517-test-installed, origin/t1517-test-installed)
Author: Adam Dinwoodie <adam@dinwoodie.org>
Date:   Sat Aug 16 11:21:18 2025 +0100

    t/t1517: mark tests that fail with GIT_TEST_INSTALLED
    
    The changes added by 39fc408562 (t/t1517: automate `git subcmd -h` tests
    outside a repository, 2025-08-08) to automatically loop over all "main"
    Git commands will, when run against an installed build using
    GIT_TEST_INSTALLED rather than the build in the build directory, include
    some extra git-gui commands that are installed by `make install`, or
    credential helpers that might be installed manually from the contrib
    directories.  These fail the test, so record them as such.
    
    Signed-off-by: Adam Dinwoodie <adam@dinwoodie.org>

diff --git a/t/t1517-outside-repo.sh b/t/t1517-outside-repo.sh
index 1c69d52c76..c824c1a25c 100755
--- a/t/t1517-outside-repo.sh
+++ b/t/t1517-outside-repo.sh
@@ -111,8 +111,11 @@ for cmd in $(git --list-cmds=main)
 do
 	cmd=${cmd%.*} # strip .sh, .perl, etc.
 	case "$cmd" in
-	archimport | cvsexportcommit | cvsimport | cvsserver | daemon | \
+	archimport | citool | credential-netrc | credential-libsecret | \
+	credential-osxkeychain | cvsexportcommit | cvsimport | cvsserver | \
+	daemon | \
 	difftool--helper | filter-branch | fsck-objects | get-tar-commit-id | \
+	gui | gui--askpass | \
 	http-backend | http-fetch | http-push | init-db | \
 	merge-octopus | merge-one-file | merge-resolve | mergetool | \
 	mktag | p4 | p4.py | pickaxe | remote-ftp | remote-ftps | \
